### PR TITLE
Added support for Mirascreen renderer

### PR DESCRIPTION
**Pull request by prescott66**

### CODE REVIEW

File: src/main/external-resources/renderers/Mirascreen.conf [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+RendererName = Mirascreen
+RendererIcon = Mirascreen.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# {friendlyName=Mirascreen BFCFA04F, manufacturer= , modelName=Mirascreen BFCFA04F, modelNumber= , modelDescription=Actions-Micro DLNA DMR Using Auto-Generated UPnP Stack, manufacturerURL=http://actions-micro.com, modelURL=http://actions-micro.com}
+# ============================================================================
+
+UserAgentSearch =
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Mirascreen.conf]: This line can be removed

File: src/main/external-resources/renderers/Mirascreen.conf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# {friendlyName=Mirascreen BFCFA04F, manufacturer= , modelName=Mirascreen BFCFA04F, modelNumber= , modelDescription=Actions-Micro DLNA DMR Using Auto-Generated UPnP Stack, manufacturerURL=http://actions-micro.com, modelURL=http://actions-micro.com}
+# ============================================================================
+
+UserAgentSearch = 
+UpnpDetailsSearch = Mirascreen BFCFA04F
+
+LoadingPriority = 3
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Mirascreen.conf]: This line can probably be removed?

File: src/main/external-resources/renderers/Mirascreen.conf [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+MediaInfo = true
+SeekByTime = true
+TranscodeVideo = MPEGTS-MPEG2-AC3
+TranscodeAudio = MP3
+DefaultVBVBufSize = true
+MaxVideoBitrateMbps = 20
+HalveBitrate = true
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Mirascreen.conf]: We generally discourage the use of HalveBitrate now so I would remove that

File: src/main/external-resources/renderers/Mirascreen.conf [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+UserAgentSearch = 
+UpnpDetailsSearch = Mirascreen BFCFA04F
+
+LoadingPriority = 3
+MediaInfo = true
+SeekByTime = true
+TranscodeVideo = MPEGTS-MPEG2-AC3
+TranscodeAudio = MP3
+DefaultVBVBufSize = true
+MaxVideoBitrateMbps = 20
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Mirascreen.conf]: Are you sure about this value? 20mbps of MPEG-2 video isn't going to give great quality. I recommend changing the TranscodeVideo line to MPEGTS-H264-AC3 instead if this limit needs to be in place

File: src/main/external-resources/renderers/Mirascreen.conf [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+# This renderer has sent the following string/s:
+#
+# {friendlyName=Mirascreen BFCFA04F, manufacturer= , modelName=Mirascreen BFCFA04F, modelNumber= , modelDescription=Actions-Micro DLNA DMR Using Auto-Generated UPnP Stack, manufacturerURL=http://actions-micro.com, modelURL=http://actions-micro.com}
+# ============================================================================
+
+UserAgentSearch = 
+UpnpDetailsSearch = Mirascreen BFCFA04F
+
+LoadingPriority = 3
+MediaInfo = true
```
Comment (SubJunk) [inline src/main/external-resources/renderers/Mirascreen.conf]: If you set MediaInfo to true without specifying any `Supported` lines, it mean everything will be transcoded. Does the renderer not support anything at all?

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 1 above)
Comment (prescott66) [reply src/main/external-resources/renderers/Mirascreen.conf]: if it is uselesss, just remove it

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 2 above)
Comment (prescott66) [reply src/main/external-resources/renderers/Mirascreen.conf]: no, without these lines, playback dont work. please keep it.

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 3 above)
Comment (prescott66) [reply src/main/external-resources/renderers/Mirascreen.conf]: okay, but remove only line with HalveBitrate

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 4 above)
Comment (prescott66) [reply src/main/external-resources/renderers/Mirascreen.conf]: with these values, playback works

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 5 above)
Comment (prescott66) [reply src/main/external-resources/renderers/Mirascreen.conf]: it just works

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 4 above)
Comment (SubJunk) [reply src/main/external-resources/renderers/Mirascreen.conf]: If I make the changes I think would be good, would you be willing to test them? Changing `MPEG2` to `H264` would give users huge video quality and bandwidth improvements

File: src/main/external-resources/renderers/Mirascreen.conf (hunk 3 above)
Comment (SubJunk) [inline src/main/external-resources/renderers/Mirascreen.conf]:
```suggestion
```

**Pull request merged**